config/node/createSchemaCustomization.js

```diff
@@ -18,8 +18,7 @@ module.exports = ({ actions }) => {
       }
 
       type Photo {
-        horizontal: File! @fileByRelativePath,
-        vertical: File! @fileByRelativePath,
+        avatar: File! @fileByRelativePath,
```

Comment:
Should we place this directly under the `Author` type? If there's just one photo anyway? CC: @gabrielpoca

Reply:
Yeah, I think so. I don't see why we need this Photo type.

Reply:
@jreis13 Will you take care of this?

Reply:
When that's done, we can review again and merge this. Then, people can check their photo live, and request a change if they want.